db/migrate/20221122214717_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly change user to author according to the project requirement so it will create author_id with each index
` t.references :author, foreign_key: { to_table: 'users' }`

Reply:
Hey @VICTOR-CHUKWU , thank you so much for your feedback, finding a solution to do that was really challenging since I still understand how **migration** works but I already solved it.